src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/DateAdd_TimeZone_Beirut.txt

```diff
@@ -0,0 +1,69 @@
+﻿// (UTC+02:00) Beirut
+#SETUP: TimeZoneInfo("Middle East Standard Time")
+
+>> DateAdd(Date(2021, 3, 26), 2)
+DateTime(2021,3,28,1,0,0,0)
+
+// When adding 2 days to this date, we arrive to an invalid date/time
+// So we calculate the next valid date/time and that's why we see this extra 1h
+>> DateAdd(Date(2021, 3, 26), 2, "days")
```

Comment:
>"days" [](http://example.com/codeflow?start=33&length=6)

Can you use the TimeUnit enumeration instead of their string value in the tests? Although they are the same, we want to "teach" people (with the tests) to use it in a more elegant way. Passing a string they may get an invalid argument, with the enumeration they won't.

- "days" - TimeUnit.Days
- "hours" - TimeUnit.Hours
- "milliseconds" - TimeUnit.Milliseconds

Reply:
Sure